Gemfile.lock

```diff
@@ -56,4 +56,4 @@ DEPENDENCIES
   uglifier
 
 BUNDLED WITH
-   1.10.6
+   2.1.4
```

Comment:
Bundler 2 requires Ruby 2.3+. Please update either the `.ruby-version` to >= 2.3 or leave the BUNDLED WITH section untouched.. new devs can't bundle with this combination.

Reply:
Thanks for pointing that out, I had manually switched to 2.3, I believe.
I suggest upgrading to a recent Ruby and Bundler, and upgrade the Gemfile.lock where possible.